pep-0570.rst

```diff
@@ -221,7 +327,7 @@ remains the same as the one for positional-or-keyword. Once
 a positional-only argument is provided with a default,
 the following positional-only and positional-or-keyword argument
```

Comment:
Should be "the following positional-only and positional-or-keyword arguments need to have defaults as well" (plural, since it applies to *all* arguments, not just the one immediately following).